doc/TBG_macros.cfg

```diff
@@ -188,6 +188,19 @@ TBG_restart="--restart"
 # Note: does currently not work with `Radiation` plugin
 TBG_softRestarts="--softRestarts 5000"
 
+# ISAAC server and rendering interval settings
+TBG_isaac="--isaac.period 1 --isaac.name !TBG_jobName --isaac.url <server_url>"
+
+# ISAAC JPEG quality (1..100). Smaller values are faster sent, but of lower quality
+TBG_isaac_quality="--isaac.quality 80"
```

Comment:
That is not the point. Afaik 99 is not really slower or faster than 80, but the file size is **much** lower with 80. Not so important for simulation -> server, but when the hd image needs to be transceived from the server to the client (in a university/conference wifi!) the chocolate blancmange hits the fan!